package.json

```diff
@@ -21,7 +21,7 @@
 		"test": "eslint ."
 	},
 	"peerDependencies": {
-		"mwbot": "^1.0.10",
+		"mwbot": "^2.0.0",
 		"wdio-mediawiki": "^1.0.0",
```

Comment:
The latest published release of [wdio-mediawiki](https://www.npmjs.com/package/wdio-mediawiki) still depends on mwbot v1 so these two peer dependencies are conflicting at the moment. I see that the update already happened in the code ([here](https://gerrit.wikimedia.org/r/c/mediawiki/core/+/680336)) but we would need a new version to be released before we can use it.

Reply:
Good point :) I've made https://gerrit.wikimedia.org/r/c/mediawiki/core/+/693400, after that's merged @zeljkofilipin could you publish the new package version on npmjs.com?